src/styles/signup-header.styles.css

```diff
@@ -0,0 +1,30 @@
+.signup-header {
+    width: 100%;
+    height: 65px;
```

Comment:
change this to use view height and view width.

exact pixel only works for one specific display. 

would like to see media queries as well for mobile reponsiveness on different size displays.

Reply:
I think maybe we should wait for media queries until we get a design from kevin, no? Do we know if he's working on those too?

Reply:
so nix using media queries for now, but still change px values